includes/emails/class-emails.php

```diff
@@ -488,6 +488,19 @@ public static function get_email_config_by_type( $type ) {
 			/** Only attempt to create the email post if wp-includes/pluggable.php is loaded. */
 			return false;
 		} else {
+			// Make sure newsletters color palette is updated with latest theme colors.
+			if ( self::supports_emails() && method_exists( '\Newspack_Newsletters', 'update_color_palette' ) ) {
+				$theme_colors = newspack_get_theme_colors();
+				\Newspack_Newsletters::update_color_palette(
+					[
+						'primary'        => $theme_colors['primary_color'],
+						'secondary'      => $theme_colors['secondary_color'],
+						'primary-text'   => $theme_colors['primary_text_color'],
+						'secondary-text' => $theme_colors['secondary_text_color'],
```

Comment:
`newspack_get_theme_colors` does not return `primary_text_color` and `secondary_text_color` keys:

https://github.com/Automattic/newspack-plugin/blob/725786b70e43ff8d8b9e3789c43cea13f70d4eb1/includes/util.php#L549-L552

So it should be:

```
					'primary-text'   => newspack_get_color_contrast( $theme_colors['primary_color'] ),
					'secondary-text' => newspack_get_color_contrast( $theme_colors['secondary_color'] ),
```

Reply:
Ah! That's right. We removed the secondary color from the email templates, so now this should only reference `primary_color` and `primary_text_color`. I've updated in https://github.com/Automattic/newspack-plugin/pull/3083/commits/e89850054f454305a5b9c41e9944414a63daee16. 

Thanks for the keen eye!